### PR TITLE
Format Python code using Black

### PR DESCRIPTION
**Pull request by Calinou**
This makes it more readable and PEP-8 compliant. If we end up adopting a formatter (which is good for consistency and avoiding bikeshedding), we should document it in a `CONTRIBUTING.md` file. Here, I ran [Black](https://github.com/ambv/black) with the default settings.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -5,430 +5,457 @@
...
+    app.config.from_pyfile("config.py")
 
 
 # Views
 
+
 @app.route("/")
 def index():
-	return app.send_static_file("index.html")
+    return app.send_static_file("index.html")
```
Comment (sfan5) [inline server.py]: no

Comment (Calinou) [reply server.py]: PEP-8 recommends using four spaces, so in the interest of following established standards, we ought to do that :smiley:

Comment (sfan5) [reply server.py]: I know that PEP-8 recommends this, but I don't like it and never will.
It's still useful to be close to the standard, so I'd just disable this behaviour in the formatter.

Comment (Calinou) [reply server.py]: [Black doesn't support tabs](https://github.com/ambv/black/issues/47), so I'm afraid that's not an optino.

**Comment by sfan5** [comment]
Closing as the formatter used here does not support our "ignorant and absurd" (authors own words) code style.

**Pull request closed without merging**